x-pack/plugins/fleet/server/services/epm/package_service.ts

```diff
@@ -80,8 +81,14 @@ export interface PackageClient {
 
   getPackage(
     packageName: string,
-    packageVersion: string
-  ): Promise<{ packageInfo: ArchivePackage; paths: string[] }>;
+    packageVersion: string,
+    options?: Parameters<typeof getPackage>['2']
```

Comment:
I'm not really familiar with this syntax, maybe it would be more readable to write the parameter type explicitly

Reply:
The `options` parameter here is forwarded to the `getPackage` function in its usage.
The original `options` type is [defined inline on the `getPackage` function definition](https://github.com/elastic/kibana/blob/main/x-pack/plugins/fleet/server/services/epm/registry/index.ts#L310), so writing explicitly the type here would de-sync the 2 types (in case `getPackage` options signature changes, it would need a manual update also here).

[Parameters](https://www.typescriptlang.org/docs/handbook/utility-types.html#parameterstype) is a native TS utility type which allows to infer the parameters type of a function and derive a new type, like in this case, so that if anything changes in the original inline type, it will reflect the change on this wrapper method signature.